### DISCUSSION TITLE
[DELIVER] Specify Availability on Apple Silicon for iOS Apps

### DISCUSSION THREAD

**Discussion by stherold** [fastlane/fastlane#17596] [Ideas]
Is it currently possible to specify the option `Proicing and Availability` -> `iPhone and iPad Apps on Apple Silicon Macs` -> `Make this app available`?

If not this would be a valuable option :)

**Comment by rogerluan** [comment]
Hey Stefan! 😁 Long time no talk!

That's a good question - I don't think we have such option in fastlane yet. @joshdholtz would this be somewhat related to the stuff you said you'd be working on this upcoming week (regarding submission information)? 😁 

Thanks for your input @stherold 😃

**Reply by andyrohan-fiserv** [reply]
I think this is taken care of in #16973

**Reply by stherold** [reply]
But I think it is not part of deliver automatically when it has been implemented in Spaceship - no?

**Reply by rogerluan** [reply]
You're right @stherold, it's just available directly through _spaceship_

**Comment by Bejaranon** [comment]
Developers will have the ability to prevent their iOS/iPadOS from appearing in the macOS App Store on the Apple Silicon computers, if they choose. I do hope that eero chooses to embrace this new opportunity though, rather than block it



[myaarpmedicare](https://www.myaarpmedicare.top/)

**Reply by stherold** [reply]
> Developers will have the ability to prevent their iOS/iPadOS from appearing in the macOS App Store on the Apple Silicon computers, if they choose. I do hope that eero chooses to embrace this new opportunity though, rather than block it

Of course this is a great opportunity. But it **never** just works out of the box. Same with universal apps. There is also the possibility to run an iPhone app on any iPad - grat. But for our app it wasn't that great without many adaptions. That's why we need a way to opt out to prevent negative feedback for apps that work great on other platforms.

**Comment by reishka** [comment]
FWIW, this no longer works through spaceship. It returns `the provided entity includes an unknown attribute: 'isOptedInToDistributeIosAppOnMacAppStore' is not an attribute on the resource 'apps'`. Can confirm it worked previously.

**Reply by rogerluan** [reply]
It seems that Apple silently dropped support to this attribute 🧐
